tests/web/classifier_service/dummy_classifier.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+from typing import Any, Dict, Hashable, Iterator, Iterable, Mapping, Sequence, Union
+
+from smqtk_classifier import (
+    ClassifyDescriptorSupervised,
+    ClassifyDescriptor
+)
+from smqtk_descriptors import DescriptorElement
+
+
+STUB_CLASSIFIER_MOD_PATH = __name__
+
+
+class DummyClassifier (ClassifyDescriptor):
+    def get_config(self) -> Dict[str, Any]: ...
+
+    @classmethod
+    def is_usable(cls) -> bool:
+        return True
+
+    def _classify_arrays(self,
+                         array_iter: Union[np.ndarray,
+                                           Iterable[np.ndarray]]) -> Iterator[Dict[Hashable, float]]:
```

Comment:
This looks less readable. Let's go with what Black would do here:
```suggestion
    def _classify_arrays(
        self,
        array_iter: Union[np.ndarray, Iterable[np.ndarray]]
    ) -> Iterator[Dict[Hashable, float]]:
```